students/taeyeon/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json,re
 
-# Create your views here.
+from django.http            import JsonResponse
+from django.views           import View
+from django.core.exceptions import ValidationError
+
+from .models                import User
+
+class UsersView(View) :
+    def post(self,request) :
+        try: 
+            data           = json.loads(request.body)
+            email          = data['email']
+            password       = data['password']
+            regex_email    = r'^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+            regex_password = r'^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
+
+            if re.match(regex_email, email) == None :
+                raise ValidationError("Email format is incorrect")
+            if re.match(regex_password, password) == None :
+                raise ValidationError("Password format is incorrect")
+            if User.objects.filter(email=email).exists():
+                raise ValidationError("Email already exists")
```

Comment:
`== None` 보다는 `not`을  사용해주세요!

```suggestion
            if not re.match(regex_email, email):
                raise ValidationError("Email format is incorrect")
                
            if not re.match(regex_password, password):
                raise ValidationError("Password format is incorrect")
                
            if User.objects.filter(email=email).exists():
                raise ValidationError("Email already exists")
```